src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -560,3 +561,110 @@ def _patch(self, crd, crd_name, cr_name, func):
                 "Failed to update {}/{}: {}".format(crd_name, cr_name, e))
 
         return "Success"
+
+    def get_ceph_image(self):
+        try:
+            api_response = self.coreV1_api.list_namespaced_pod(self.rook_env.namespace,
+                                                               label_selector="app=rook-ceph-mon",
+                                                               timeout_seconds=10)
+            if api_response.items:
+                return api_response.items[-1].spec.containers[0].image
+            else:
+                raise orchestrator.OrchestratorError(
+                        "Error getting ceph image. Cluster without monitors")
+        except ApiException as e:
+            log.error("Error getting ceph image: {}".format(e))
+
+    def blink_light(self, ident_fault, on, locs):
+        # type: (str, bool, List[DeviceLightLoc]) -> string
+        operation_id = str(hash(locs))
+        message = ""
+
+        # job definition
+        job_metadata = client.V1ObjectMeta(name=operation_id,
+                                           namespace= self.rook_env.namespace,
+                                           labels={"ident": operation_id})
+        pod_metadata = client.V1ObjectMeta(labels={"ident": operation_id})
+        pod_container = client.V1Container(name="ceph-lsmcli-command",
+                                           image=self.get_ceph_image(),
+                                           command=["lsmcli",],
+                                           args=['local-disk-%s-led-%s' % (ident_fault,'on' if on else 'off'),
+                                                 '--path', locs.path or locs.dev,])
```

Comment:
`rook/rook_cluster.py:592: error: "List[Any]" has no attribute "path"`

Reply:
yeah, removing the type annotation silenced the error message, but `locs` is still of type `List[DeviceLightLoc]`, which still doesn't have a `path` attribute! The type annotations were correct, just the code is wrong.